### ISSUE TITLE
Nearly all Useful Links in the primary readme are 404s

### ISSUE THREAD

**Issue by blakeplusplus** [payload-bot/payload-neo#200]
**Describe the bug**
See title. Location of links are [here](https://github.com/payload-bot/payload-neo?tab=readme-ov-file#useful-links).
The Invite link leads to a 404.
The Discord link leads to a 404.
The Documentation link leads to a 404.
The Feature Board is listed as private; I cannot view it without logging in. A good alternative is Miro if you are unable to make it public.
The Changelog link leads to a 404.

**To Reproduce**
Go to each of the links in a private/incognito browser session.

**Comment by c43721** [comment]
Yup, I had a PR in another top secret project that fixed that. Just need to clone them over. Thanks!
